ansible/configs/ibm-rhoic/software.yml

```diff
@@ -255,6 +255,10 @@
         var: r_existing_rhoic
         verbosity: 2
 
+    - set_fact:
+        email: "no-valid-email-set"
```

Comment:
you can get a valid email address from `{{ email }}`

Reply:
Added a name that describes when this runs as per our conversation.